src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.h

```diff
@@ -5,13 +5,20 @@
 
 struct ItemEtcetera;
 
+#define ITEMETCETERA_GET_FF(thisx) ((thisx)->params & 0xFF)
+#define ITEMETCETERA_GET_TREASURE(thisx) (((thisx)->params >> 8) & 0x1F)
+
 typedef void (*ItemEtceteraActionFunc)(struct ItemEtcetera*, GlobalContext*);
 
 typedef struct ItemEtcetera {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x10];
-    /* 0x0154 */ ItemEtceteraActionFunc actionFunc;
-} ItemEtcetera; // size = 0x158
+    /* 0x0144 */ ItemEtceteraActionFunc funcSetup;
+    /* 0x148 */ s16 unk_148;
+    /* 0x14A */ s16 unk_14A;
+    /* 0x14C */ u8 unk_14C;
+    /* 0x14D */ ActorFunc drawFunc;
+    /* 0x0151 */ ItemEtceteraActionFunc actionFunc;
```

Comment:
```suggestion
    /* 0x150 */ ActorFunc drawFunc;
    /* 0x154 */ ItemEtceteraActionFunc actionFunc;
```
These are still going to be at the offsets specified before, even with `char pad_14D[3]` removed.